src/nft/components/profile/list/FloatingBanner.tsx

```diff
@@ -0,0 +1,112 @@
+import { buttonHoverState } from 'components/Common'
+import { getTotalEthValue } from 'nft/components/bag/profile/utils'
+import { useSellAsset } from 'nft/hooks'
+import { Listing, WalletAsset } from 'nft/types'
+import { formatUsdPrice } from 'nft/utils/currency'
+import { fetchPrice } from 'nft/utils/fetchPrice'
+import { useEffect, useMemo, useState } from 'react'
+import styled from 'styled-components/macro'
+import { ThemedText } from 'theme'
+
+const FloatingBannerGradientContainer = styled.div`
+  display: none;
+  position: fixed;
+
+  justify-content: center;
+  bottom: 0;
+  background: linear-gradient(180deg, rgba(0, 0, 0, 0) 0%, #000000 100%);
+  width: calc(100% - 360px);
+  left: calc((100% - 360px) / 2);
+  transform: translateX(-50%);
+  padding-top: 48px;
+  padding-bottom: 48px;
+
+  @media only screen and (min-width: ${({ theme }) => `${theme.breakpoint.lg}px`}) {
+    display: flex;
+  }
+`
+
+const FloatingBannerContainer = styled.div`
+  background-color: ${({ theme }) => theme.backgroundSurface};
+  border-radius: 20px;
+  display: flex;
+  width: calc(100% - 100px);
+  justify-content: space-between;
+  align-items: center;
+  padding: 24px 48px;
+`
+
+const Content = styled.div`
+  display: flex;
+  gap: 16px;
+  align-items: center;
+`
+
+const ListButton = styled.button`
+  position: relative;
+  background-color: ${({ theme }) => theme.accentAction};
+  color: ${({ theme }) => theme.white};
+  border: none;
+  font-weight: 500;
+  border-radius: 12px;
+  padding: 10px 12px;
+  text-align: center;
+  font-size: 16px;
+
+  ${buttonHoverState}
+
+  &:disabled {
+    opacity: 0.3;
+  }
+`
+
+export const FloatingBanner = () => {
+  const sellAssets = useSellAsset((state) => state.sellAssets)
+  const totalEthListingValue = useMemo(() => getTotalEthValue(sellAssets), [sellAssets])
+  const [ethConversion, setEthConversion] = useState(3000)
+
+  useEffect(() => {
+    fetchPrice().then((price) => {
+      setEthConversion(price ?? 0)
+    })
+  }, [])
```

Comment:
Can we use the simpler pattern of `const { data: ethConversion } = useQuery(['fetchPrice', {}], () => fetchPrice(), {})` that is used in places such as `src/nft/components/details/AssetPriceDetails.tsx`